test/python/sdpa/random_config.py

```
@@ -9,8 +9,14 @@
 # fmt: off
 
 def generate_test_seeds(*, num_tests, rng_seed):
+    # Env overrides for focused/expanded sweeps without editing each suite:
+    #   MHAS_NUM_TESTS   - cap (or expand) the number of configs (e.g. small subset for sanitizer runs)
+    #   MHAS_SEED_OFFSET - shift the geometry RNG seed to explore a fresh set of configs
+    import os
+    rng_seed = rng_seed + int(os.environ.get("MHAS_SEED_OFFSET", "0"))
+    n = int(os.environ.get("MHAS_NUM_TESTS", "0")) or num_tests
     rng = random.Random(rng_seed)
```

Ruff (0.15.18)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)